cellprofiler/__main__.py

```diff
@@ -81,7 +80,11 @@ def main(args=None):
         print_groups(options.print_groups_file)
 
     if options.batch_commands_file is not None:
-        get_batch_commands(options.batch_commands_file)
+        try:
+            nr_per_batch = int(options.images_per_batch)
+        except:
```

Comment:
You should catch an explicit exception rather than all exceptions. Are you trying to test whether `images_per_batch` is set? If so, you don't need this since you provided a default.